documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -433,7 +433,15 @@ For more information about the `livenessProbe` and `readinessProbe` options, see
 == `metricsConfig`
 
 Use the `metricsConfig` property to enable and configure Prometheus metrics.
+Strimzi provides support for Prometheus JMX Exporter and Strimzi Metrics Reporter. Only one of these can be selected at any given time.
 
+When metrics are enabled, they are exposed on port 9404.
+
+When the `metricsConfig` (or deprecated `metrics`) property is not defined in the resource, the Prometheus metrics are disabled.
+
+For more information about setting up and deploying Prometheus and Grafana, see link:{BookURLDeploying}#assembly-metrics-str[Introducing Metrics to Kafka^].
+
+.Using Prometheus JMX Exporter
 The `metricsConfig` property contains a reference to a ConfigMap that has additional configurations for the {JMXExporter}.
```

Comment:
```suggestion
The `metricsConfig` property contains a reference to a `ConfigMap` that has additional configurations for the {JMXExporter}.
```

Reply:
Can you update the ConfigMap to `ConfigMap` on line 448 too? Thanks!

Reply:
Fixed